R/model.R

```diff
@@ -263,6 +263,8 @@ CmdStanModel$set("public", name = "compile", value = compile_method)
 #'     thin = NULL,
 #'     max_depth = NULL,
 #'     metric = NULL,
+#'     inv_metric = NULL,
+#'     metric_file = NULL,
```

Comment:
is the order of these three arguments intentional? If not then I guess I slightly prefer keeping the two actual CmdStan arguments together in the signature, so that would mean either `inv_metric` before or after the other two. Or is the idea to encourage using `inv_metric` instead of `metric_file`? (that could be a good idea i guess)

Reply:
Switched